internal/pkg/cli/app_delete.go

```diff
@@ -293,9 +293,9 @@ func (o *deleteAppOpts) deleteAppConfigs() error {
 
 func (o *deleteAppOpts) deleteWs() error {
 	o.spinner.Start(deleteAppWsStartMsg)
-	if err := o.ws.DeleteAll(); err != nil {
-		o.spinner.Stop(log.Serror("Error deleting local workspace folder."))
-		return fmt.Errorf("delete workspace: %w", err)
+	if err := o.ws.DeleteWorkspaceFile(); err != nil {
+		o.spinner.Stop(log.Serrorf("Error deleting %s file.", workspace.WorkspaceSummaryFileName))
```

Comment:
nit: can you rename the variable to `SummaryFileName` so that it reads `workspace.SummaryFileName` instead of `workspace.WorkspaceSummaryFileName`?